demisto_sdk/commands/validate/validators/PA_validators/PA114_pack_metadata_version_should_be_raised.py

```diff
@@ -0,0 +1,184 @@
+        "(major|minor|revision|documentation)` for a specific pack and version."
+    )
+
+    @staticmethod
+    def should_bump_is_metadata(content_item: ContentTypes):
+        """Return if the version should be bumped as a result of the input content item.
+        Args:
+            content_item (ContentTypes): The content item for which to decide.
+
+        Return:
+            ((Boolean) True if should bump due to change False otherwise,
+             (Boolean) True if the content item is the pack_metadata False otherwise)
+        """
+        content_item_type = find_type(str(content_item.path))  # type: ignore[union-attr]
+        is_metadata = False
+
+        if (
+            not content_item_type
+            and content_item.content_type.value == Pack.content_type.value  # type: ignore[union-attr]
+        ):
+            content_item_type = FileType.METADATA
+            is_metadata = True
+
+        if content_item_type not in SKIP_RELEASE_NOTES_FOR_TYPES:
+            if content_item_type != FileType.METADATA:
+                return True, False
+            else:
+                is_metadata = True
+                old_dict = content_item.old_base_content_object.to_dict()  # type: ignore[union-attr]
+                current_dict = content_item.to_dict()  # type: ignore[union-attr]
+                for field in PACK_METADATA_REQUIRE_RN_FIELDS:
+                    if old_dict.get(field) != current_dict.get(field):
+                        return True, True
+        return False, is_metadata
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> typing.List[ValidationResult]:
+        validation_results = []
+        content_packs = {}
+        content_packs_ids_to_bump = []
+        # Go over all the content items
+        for content_item in content_items:
+            should_bump, is_metadata = self.should_bump_is_metadata(content_item)
+            if should_bump:
+                pack_id = content_item.pack_id  # type: ignore[union-attr]
+                if pack_id not in content_packs_ids_to_bump:
+                    # Collect content pack ids that should be bumped.
+                    content_packs_ids_to_bump.append(pack_id)
+            if is_metadata:
+                # Collect content metadata items and link them to their pack ids.
+                content_packs[content_item.pack_id] = content_item  # type: ignore[union-attr]
+
+        # Go over all the pack ids that need to be bumped.
+        for pack_id in content_packs_ids_to_bump:
+            # Access them via the dict that was created earlier
+            pack = content_packs[pack_id]
+            # Check if their old version >= current version
+            old_version = pack.old_base_content_object.current_version  # type: ignore[union-attr]
+            current_version = pack.current_version  # type: ignore[union-attr]
+            if current_version and Version(old_version) >= Version(current_version):
```

Comment:
maybe we can do this check first? might be quicker than iterating on all items especially after we group by pack

Reply:
We don't know if we should bump yet before / we are not sure we have collected the metadata content item yet. Not sure I fully understand your comment.